sixtyfps_runtime/corelib/flickable.rs

```diff
@@ -75,13 +75,12 @@ impl FlickableData {
                 }
             }
             MouseEvent::MouseMoved { pos } => {
-                if inner.capture_events
-                    || inner.pressed_time.map_or(false, |pressed_time| {
-                        crate::animations::current_tick() - pressed_time < DURATION_THRESHOLD
-                            && (pos - inner.pressed_pos).square_length()
-                                > DISTANCE_THRESHOLD * DISTANCE_THRESHOLD
-                    })
-                {
+                let outside_of_thresholds = inner.pressed_time.map_or(false, |pressed_time| {
```

Comment:
We don't want to call current_tick() if we don't have too. So we should keep the capture_events condition before.